Alfie/Alfie/Views/DebugMenu/FeatureToggle/FeatureToggleViewModel.swift

```diff
@@ -0,0 +1,51 @@
+import Combine
+import Models
+import SwiftUI
+
+final class FeatureToggleViewModel: FeatureToggleViewModelProtocol {
+    @Published private(set) var features: [(feature: String, isEnabled: Bool)] = []
+    @Published var isDebugConfigurationEnabled = false
+
+    private let provider: any DebugConfigurationProviderProtocol
+    private var store: Set<AnyCancellable> = []
```

Comment:
Maybe it could be a single subscription?
```suggestion
    private var subscription: AnyCancellable?
```